src/helpers/FormioLoader/FormioLoader.js

```diff
@@ -123,11 +122,6 @@ const initFormioInstance = (elem, opts) => {
     if (typeof opts.createFormCallback === "function") {
       // call custom callback hook
       opts.createFormCallback(callbackProps);
-    } else {
```

Comment:
Q: @qld-gov-au/product-management  and others: What occurs if you start the form and then have a link in a 'hint' or at the bottom next to the submit button but not submitted yet.

Without this will it :
a: lose form content and you need to start again.
b: not lose form content and the back button will bring it back if using same 'window'.

When doing a low distraction form, it should really not allow you to use data when mid way through a form. 

Q2: Should this be enforced or only 'enabled' if the form has started? (i.e. put in logic to see if the form is 'dirty' in formio.js eyes in that the use has begun.)

Reply:
have explained to manjeet and that make sense
i'll revert this part and using this PR for sync the main branch